packages/react-composites/src/composites/MeetingComposite/state/MeetingState.ts

```diff
@@ -1,8 +1,57 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import { CallClientState, CallState, RemoteParticipantState } from '@internal/calling-stateful-client';
+import { ChatClientState, ChatThreadClientState } from '@internal/chat-stateful-client';
+import { ChatParticipant } from '@azure/communication-chat';
+import { CallEndReason } from '@azure/communication-calling';
+import { CommunicationIdentifier } from '@azure/communication-common';
+
+export type NonApplicableClientState =
+  | 'participants'
+  | 'userId'
+  | 'calls'
+  | 'callsEnded'
+  | 'incomingCalls'
+  | 'incomingCallsEnded';
+
 /**
- * This will be a combination of Chat and Call state where props
- * like Participants are combined into one.
+ * Meeting State is a combination of Stateful Chat and Stateful Calling clients.
+ * Stateful items like Participants that apply to both calling and chat are intelligently
+ * combined into one to suit the purpose of a Meeting.
  */
-export type MeetingState = unknown;
+export interface MeetingClientState
```

Comment:
Similar to my comment on #678: Explicitly allowlist the state constituents with `Pick` rather than filtering with `Omit`. Same reasoning as there.

Reply:
Updated to Pick over Omit